app/guid-file/styles.scss

```diff
@@ -51,7 +51,7 @@
             background-color: $color-white;
             color: $color-gray;
         }
-        background-color: $color-bg-gray-light;
```

Comment:
color-gray-light is darner than color-bg-gray-light, so did you verify any potential text a11y issues?